### ISSUE TITLE
17.12.0-ce-mac47 does not finish launching

### ISSUE THREAD

**Issue by hakamadare** [docker/for-mac#2452]
Diagnostic ID: 44BAF7CF-03F6-4621-AA11-FAD1DA2CA04C

### Expected behavior

1. Launch Docker.app.
2. Docker.app launches, menuling appears.
3. Menuling application stops, docker daemon is running.

### Actual behavior

1. Launch Docker.app.
2. Docker.app launches, menuling appears.
3. Error dialog: "Fatal Error com.docker.supervisor failed to start
Exit code 1"

### Information

  - Full output of the diagnostics from "Diagnose & Feedback" in the menu

```
Docker for Mac: version: 17.12.0-ce-mac47 (72b93a017350990850ddc37cd341bd16fce3e911)
macOS: version 10.13.2 (build: 17C88)
logs: /tmp/44BAF7CF-03F6-4621-AA11-FAD1DA2CA04C/20180117-094834.tar.gz
failure: com.docker.vpnkit is not running
[OK]     db.git
[OK]     vmnetd
[OK]     dns
[ERROR]  driver.amd64-linux
         com.docker.driver.amd64-linux is not running
[OK]     virtualization VT-X
[OK]     app
[ERROR]  moby
         /Users/rk/Library/Containers/com.docker.docker/Data/com.docker.driver.amd64-linux/console-ring does not exist
[OK]     system
[OK]     moby-syslog
[OK]     kubernetes
[OK]     env
[OK]     virtualization kern.hv_support
[ERROR]  slirp
         Unexpected error ((Failure
  "Error connecting socket to 9p endpoint unix:/Users/rk/Library/Containers/com.docker.docker/Data/s51: Unix.Unix_error(Unix.ECONNREFUSED, \"connect\", \"\")")) connecting to /Users/rk/Library/Containers/com.docker.docker/Data/s51
         com.docker.vpnkit is not running
         slirp check failed with: Unix.Unix_error(Unix.ECONNREFUSED, "connect", "")
[ERROR]  osxfs
         com.docker.osxfs is not running
[OK]     moby-console
[OK]     logs
[ERROR]  docker-cli
         Connection refused (ECONNREFUSED) connecting to /var/run/docker.sock: check if service is running
         Connection refused (ECONNREFUSED) connecting to /Users/rk/Library/Containers/com.docker.docker/Data/s60: check if service is running
         docker ps failed
[OK]     menubar
[OK]     disk
```

### Steps to reproduce the behavior

  1. Launch Docker.app

### Troubleshooting steps I have tried

1. Reset Docker to factory defaults, relaunch
2. `brew cask zap docker && brew cask install docker`

**Comment by hakamadare** [comment]
#2453 and #2447 look like duplicates of this issue, perhaps?

**Comment by hakamadare** [comment]
rolling back to 17.09.0-ce-mac35 (19611) resulted in a Docker.app that would launch successfully and a running docker daemon.

**Comment by akimd** [comment]
Hi!

Could you please try this modified version of Docker 17.12? Ignore upgrades it might propose. Hopefully it should fix your issue. If it does, we will soon release an official stable image which includes the fix. Once you tested it, be sure to reinstall a "real" stable release.

https://download-stage.docker.com/mac/pr/21923/Docker.dmg

**Comment by akimd** [comment]
Could you please check if the following image fixes your problems?  You might need to reset to factory defaults...

https://download-stage.docker.com/mac/pr/21949/Docker.dmg

This is a test build. Ignore any update it might propose, and after testing, reinstall an Edge or Stable version to get proper updates again. Thanks!

**Comment by gtardif** [comment]
We just released 17.12.0-ce-mac49 on the Stable channel that should fix this. (and 18.01 on the Edge channel also fixes this)

**Comment by hakamadare** [comment]
confirmed, 17.12.0-ce-mac49 works for me.  thank you!
